### PR TITLE
docs: translate `react-dom` -> Components -> Common

### LINKED ISSUE #422: Common (e.g. `<div>`)

### PR DESCRIPTION
**Pull request by FelixFern**
Closes #422 

Translation done ready for review https://id.react.dev/reference/react-dom/components/common page

Reference: https://github.com/reactjs/id.react.dev/issues/330

### CODE REVIEW

File: src/content/reference/react-dom/components/common.md [hunk 1]
```diff
@@ -1,95 +1,96 @@
 ---
-title: "Common components (e.g. <div>)"
+title: "Komponen umum (e.g. <div>)"
 ---
 
 <Intro>
 
-All built-in browser components, such as [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), support some common props and events.
+Semua komponen bawaan dari sebuah peramban web (*browser*), seperti [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), mendukung beberapa *props* umum dan *event*.
```
Comment (resir014) [inline src/content/reference/react-dom/components/common.md:7]:
```suggestion
Semua komponen bawaan dari sebuah peramban web (*browser*), seperti [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), mendukung beberapa *props* dan *event* umum. 
```

Comment (FelixFern) [reply src/content/reference/react-dom/components/common.md:7]: Noted and fixed, thank you for the review

File: src/content/reference/react-dom/components/common.md [hunk 2]
```diff
@@ -985,12 +988,12 @@ textarea { display: block; margin-top: 5px; margin-bottom: 10px; }
 
 </Sandpack>
 
-To see why rendering arbitrary HTML is dangerous, replace the code above with this:
+Untuk mengetahui mengapa merender HTML sewenang-wenang itu berbahaya, ganti kode di atas dengan ini:
```
Comment (r17x) [inline src/content/reference/react-dom/components/common.md:991]:
```suggestion
Untuk mengetahui mengapa me-*render* HTML sewenang-wenang itu berbahaya, ganti kode di atas dengan ini:
```

**Review by r17x — CHANGES_REQUESTED** [review]

File: src/content/reference/react-dom/components/common.md (hunk 2 above)
Comment (FelixFern) [reply src/content/reference/react-dom/components/common.md]: noted and fixed, thank you for the review

**Comment by FelixFern** [comment]
Hi @resir014 @zainfathoni @r17x , this PR is ready for review, please check :D 
Thanks!

File: src/content/reference/react-dom/components/common.md [hunk 3]
```diff
@@ -1,327 +1,329 @@
 ---
-title: "Common components (e.g. <div>)"
+title: "Komponen umum (e.g. <div>)"
```
Comment (resir014) [inline src/content/reference/react-dom/components/common.md:2]:
```suggestion
title: "Komponen umum (cth. <div>)"
```

`e.g.` -> `cth.`

**Review by resir014 — CHANGES_REQUESTED** [review]
One minor change.

**Review by r17x — APPROVED** [review]

File: src/content/reference/react-dom/components/common.md (hunk 3 above)
Comment (FelixFern) [reply src/content/reference/react-dom/components/common.md]: thank you for the review, resolved

**Review by resir014 — APPROVED** [review]

**Comment by resir014** [comment]
@FelixFern All good, please resolve the merge conflicts before we can merge this.

**Comment by FelixFern** [comment]
conflict resolve @resir014

File: src/content/reference/react-dom/components/common.md [hunk 4]
```diff
@@ -1,327 +1,328 @@
...
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
-### Common components (e.g. `<div>`) {/*common*/}
+### Komponen umum (e.g. `<div>`) {/*common*/}
 
 ```js
-<div className="wrapper">Some content</div>
+<div className="wrapper">beberapa konten</div>
```
Comment (resir014) [inline src/content/reference/react-dom/components/common.md:20]:
```suggestion
<div className="wrapper">Konten sembarang</div>
```

**Pull request merged**